src/user/supla_esp_cfg.h

```diff
@@ -50,6 +50,7 @@ typedef struct {
     char InputCfgTriggerOff;
 
     char FirmwareUpdate;
+    char ZeroInitialEnergy;
```

Comment:
You can not put it in this place.
This causes damage to existing configurations.
Add this just before "char zero [200]" if you must.

Please do the counter resetting procedure the same way here https://github.com/SUPLA/supla-espressif-esp/blob/master/src/include/board/impulse_counter.c#L435